src/runtime/plan/planificator.ts

```diff
@@ -191,14 +177,15 @@ export class Planificator {
 
   async _storeSearch(): Promise<void> {
     const values = await this.searchStore['get']() || [];
+    const arcKey = this.arc.storageProviderFactory.parseStringAsKey(this.arc.storageKey).arckey;
```

Comment:
Doesn't `arckey` need to be `arckey()`? It's defined as a standard method on the key classes, not a getter.

Reply:
good catch, thanks!